lib/fs.js

```diff
@@ -2392,6 +2403,7 @@ ReadStream.prototype.close = function(cb) {
 };
 
 fs.createWriteStream = function(path, options) {
+  path = getPathFromURL(path, true);
```

Comment:
That should actually be the wrong spot to test for this. It should be tested in `WriteStream` (the same applies to `ReadStream`).